package.json

```diff
@@ -48,11 +48,11 @@
     "@rollup/plugin-node-resolve": "7.x.x",
     "@testing-library/react-hooks": "3.x.x",
     "coveralls": "3.x.x",
-    "immer": "6.x.x",
+    "immer": "8.x.x",
     "normalizr": "3.x.x",
     "npm-run-all": "4.x.x",
-    "react": "^16.8.x",
-    "react-test-renderer": "^16.8.x",
+    "react": "17.x.x",
+    "react-test-renderer": "16.8.x",
```

Comment:
Hey @wswoodruff react-test-renderer v16 is meant to work with react v16.  I believe this should be on v17.